data/language/english_uk.txt

```diff
@@ -4094,6 +4094,7 @@ STR_5786    :Invalid group name
 STR_5787    :{COMMA32} players online
 STR_5788    :Default inspection interval:
 STR_5789    :Disable lightning effect
+STR_5799    :{OUTLINE}{YELLOW}{STRING}{WHITE} was moved to group {YELLOW}{STRING}.
```

Comment:
This string now collides with the merged #3254 & later on with PR #3192 
Should be changed to STR_5804